substrate/frame/revive/src/tests/sol.rs

```diff
@@ -30,11 +30,44 @@ use frame_support::traits::fungible::Mutate;
 use pallet_revive_fixtures::{compile_module_with_type, Fibonacci, FixtureType};
 use pretty_assertions::assert_eq;
 
+use revm::bytecode::opcode::*;
+
+mod arithmetic;
+mod bitwise;
 mod block_info;
 mod contract;
+mod control;
+mod host;
+mod memory;
+mod stack;
 mod system;
 mod tx_info;
 
+pub fn make_evm_bytecode_from_runtime_code(runtime_code: &Vec<u8>) -> Vec<u8> {
```

Comment:
```suggestion
fn make_initcode_from_runtime_code(runtime_code: &Vec<u8>) -> Vec<u8> {
```

Both of them are bytecodes. Also, this is only used in sub modules, right? So no need for `pub`.